migration/sql-files/056-categories.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE categories (
+	created_at timestamp with time zone,
+	updated_at timestamp with time zone,
+	deleted_at timestamp with time zone,
+	id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+	name text NOT NULL
+);
+
+CREATE INDEX categories_id_index ON categories (id);
+CREATE UNIQUE INDEX categories_name_idx ON categories (name) WHERE deleted_at IS NULL;
+
+CREATE TABLE workitemtype_categories (
+	created_at timestamp with time zone,
+	updated_at timestamp with time zone,
+	deleted_at timestamp with time zone,
+	id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+	category_id uuid NOT NULL,
+	workitemtype_id uuid NOT NULL
+);
+
+CREATE INDEX workitemtype_categories_id_idx ON workitemtype_categories (id);
+CREATE UNIQUE INDEX workitemtype_categories_idx ON workitemtype_categories (category_id, workitemtype_id) WHERE deleted_at IS NULL;
+ALTER TABLE workitemtype_categories ADD CONSTRAINT category_id_categories_id_fk FOREIGN KEY (category_id) REFERENCES categories (id) ON DELETE CASCADE;
```

Comment:
Do you think we could add migration tests for this change?

Sample:
https://github.com/almighty/almighty-core/commit/b76505860a5d1dfd94e0b67eea3c9c427fd1e159#diff-8300fcfb905de8543555d4fff40a7c2fR227